mappings/net/minecraft/world/biome/OverworldBiomeCreator.mapping

```diff
@@ -11,9 +11,17 @@ CLASS net/minecraft/unmapped/C_zulhefzk net/minecraft/world/biome/OverworldBiome
 		ARG 4 sparse
 		ARG 5 unmodified
 		ARG 6 spawnSettings
+		ARG 7 music
 	METHOD m_czonisgu createMangroveSwamp (Lnet/minecraft/unmapped/C_pzdchrcy;Lnet/minecraft/unmapped/C_pzdchrcy;)Lnet/minecraft/unmapped/C_orlkpefs;
 		ARG 0 features
 		ARG 1 carvers
+	METHOD m_eepimsva create (ZFFLnet/minecraft/unmapped/C_ypgfhxmz$C_ooyakwqo;Lnet/minecraft/unmapped/C_qbjpvmph$C_bwxzcdhs;Lnet/minecraft/unmapped/C_usodapnd;)Lnet/minecraft/unmapped/C_orlkpefs;
+		ARG 0 hasPrecipitation
+		ARG 1 temperate
+		ARG 2 downfall
+		ARG 3 spawnSettings
+		ARG 4 generationSettings
```

Comment:
oh, shouldn't these be specifically be suffixed with "builder"?

Reply:
I personally wouldn't, but if someone else says yes I'll go for it